extra/release_tool.py

```diff
@@ -416,6 +418,7 @@ def filter_docker_compose_files_list(list, version):
         "docker-compose.auditlogs.yml",
         "docker-compose.connect.yml",
         "docker-compose.config.yml",
+        "docker-compose.monitor.yml",
```

Comment:
This line (only this line!) will require cherry-pick to maintained integration branches: 2.7.x, 2.6.x and (just in case) 2.4.x. Please modify it in a separate commit so that is easy to port.

Reply:
fixed in https://github.com/mendersoftware/integration/pull/1349/commits/be088cc9de7462b87ce905ff8e3c3cb41b321ee7

Reply:
https://github.com/mendersoftware/integration/pull/1359
https://github.com/mendersoftware/integration/pull/1358
https://github.com/mendersoftware/integration/pull/1357